.travis.yml

```diff
@@ -0,0 +1,57 @@
+sudo: required
```

Comment:
Travis CI is no longer free to open source projects so we use GitHub Actions instead.

Reply:
Yep, figured that would be where I need to do most of the work.

Honestly, I thought I told git to remove that file during the rebase. But, perhaps it's good to have for reference until the PR is ready for re-review. It will definitely be gone by then.